packages/account/src/Sections/Security/ConnectedApps/connected-apps-revoke-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Button, Icon, Modal, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TConnectedAppsRevokeModalProps = {
+    handleRevokeAccess: () => void;
+    handleToggleModal: (app_id?: number | null) => void;
+    is_modal_open: boolean;
+};
+
+const ConnectedAppsRevokeModal = ({
+    handleRevokeAccess,
+    handleToggleModal,
+    is_modal_open,
+}: TConnectedAppsRevokeModalProps) => (
+    <Modal is_open={is_modal_open} className='connected-apps' width='44rem'>
+        <Modal.Body>
+            <div className='connected-apps-modal--wrapper'>
+                <div className='connected-apps-modal--icon'>
+                    <Icon icon='IcAccountTrashCan' size={128} />
+                    <Text as='p' color='prominent' weight='bold'>
+                        <Localize i18n_default_text='Confirm revoke access?' />
+                    </Text>
+                </div>
+                <div className='connected-apps-modal--buttons'>
+                    <Button large secondary onClick={() => handleToggleModal()}>
```

Comment:
**NIT:** This can be written as `<Button large secondary onClick={handleToggleModal}>`